presto-docs/src/main/sphinx/develop/presto-native.rst

```diff
@@ -0,0 +1,96 @@
+===========================
+Presto Native - Prestissimo
+===========================
+
+Prestissimo is a C++ drop-in replacement for Presto workers based on the Velox
+library. It implements the same RESTful API as Java workers using the Proxygen
+C++ HTTP framework. Since communication with the Java coordinator and across
+workers in only done via the REST APIs, Prestissimo does not use JNI and it is
```

Comment:
Suggest "workers is only done", not "in". 
Also, suggest not using Latin terms such as via. Consider alternatives such as with, through, or by using. (Reference: see "via" in [Gitlab's documentation style guide](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html).)